packages/shared/src/hooks/useDisableFilterAlert.ts

```diff
@@ -0,0 +1,26 @@
+import { useEffect } from 'react';
+import request from 'graphql-request';
+import { UPDATE_ALERTS } from '../graphql/alerts';
+import { apiUrl } from '../lib/config';
+import { FeedSettings } from '../graphql/feedSettings';
+import useAlertContext from './useAlertContext';
+
+export default function useDisableFilterAlert(
+  feedSettings: FeedSettings,
+): void {
+  const { alerts, setAlerts } = useAlertContext();
+
+  useEffect(() => {
+    if (
+      alerts?.filter &&
+      (feedSettings?.includeTags?.length ||
+        feedSettings?.blockedTags?.length ||
+        feedSettings?.excludeSources?.length ||
+        feedSettings?.advancedSettings?.length)
+    ) {
```

Comment:
Does it cover the case where I already have my feed configured and I have the red dot because it's the first I see the new sidebar?

Reply:
This is that part. If the alerts.filter is true and it has any pre-configured values inside the `feedSettings` it will remove the red dot. This will be triggered on a load of the component where this was used.